### PR TITLE
Add instructions for running x86 QEMU for PVH Domain-0

### PR DESCRIPTION
**Pull request by andr2000**
Signed-off-by: Oleksandr Andrushchenko <oleksandr_andrushchenko@epam.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,3 +9,5 @@ We are Xen Troops!
 Current documentation is located in [manifest wiki](https://github.com/xen-troops/manifests/wiki), but it will be moved to this site eventually.
 
 Product descriptions can be found [here](products)
+
+How-to build and run PVH Dom0 on x86 QEMU can be founf [here](x86_pvh_dom0.md)
```
Comment (rshym) [inline README.md]: s/founf/found/

File: x86_pvh_dom0.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
+# X86 Xen on QEMU (with PVH Dom0 support)
+>These instructions allow running a complete QEMU based environment for running X86 Xen.
+
+## Compile QEMU binaries
+> You might want trying the QEMU provided by your OS distibution before building your own
```
Comment (rshym) [inline x86_pvh_dom0.md]: s/distibution/distribution/

File: x86_pvh_dom0.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+ln -s debian-10-nocloud-amd64-20210721-710.qcow2 domu.qcow2
+```
+## Donwload and build DomU's Linux kernel
+```sh
+git clone https://github.com/torvalds/linux.git
+git checkout v5.10
+make x86_64_defconfig
+make xen.config
+```
+Update the configuration to built in the network and block device drivers:
```
Comment (rshym) [inline x86_pvh_dom0.md]: "to built"? It could be good to reword this part.

Comment (andr2000) [reply x86_pvh_dom0.md]: Ok

File: x86_pvh_dom0.md (hunk 2 above)
Comment (andr2000) [reply x86_pvh_dom0.md]: thanks

File: README.md (hunk 1 above)
Comment (andr2000) [reply README.md]: thanks

**Review by rshym — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -9,3 +9,5 @@ We are Xen Troops!
 Current documentation is located in [manifest wiki](https://github.com/xen-troops/manifests/wiki), but it will be moved to this site eventually.
 
 Product descriptions can be found [here](products)
+
+How-to build and run PVH Dom0 on x86 QEMU can be found [here](x86_pvh_dom0.md)
```
Comment (lorc) [inline README.md:13]: Are you sure you need `.md` extension? Previous one (L11) has no extension.

File: x86_pvh_dom0.md [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+
+Update the kernel to more recent one:
+```sh
+apt update
+apt-get install linux-image-5.10.0-0.bpo.8-amd64 linux-headers-5.10.0-0.bpo.8-amd64 
+```
+
+Create a new user:
+```sh
+adduser a2k
```
Comment (lorc) [inline x86_pvh_dom0.md:64]: Maybe `s/a2k/username`?

File: x86_pvh_dom0.md [hunk 6]
```diff
@@ -0,0 +1,175 @@
...
+```
+## Donwload and build DomU's Linux kernel
+```sh
+git clone https://github.com/torvalds/linux.git
+git checkout v5.10
+make x86_64_defconfig
+make xen.config
+```
+Update the configuration to make the network and block device drivers be built in into the kernel:
+```s
```
Comment (lorc) [inline x86_pvh_dom0.md:147]: I believe it should be `sh`

**Comment by rshym** [comment]
PR is replaced by the #9, so will be closed as agreed with the author.

**Pull request closed without merging**